homeassistant/components/tuya/sensor.py

```diff
@@ -516,6 +516,31 @@ class TuyaSensorEntityDescription(SensorEntityDescription):
         ),
         *BATTERY_SENSORS,
     ),
+    # Smart Water Timer
+    "sfkzq": (
+        TuyaSensorEntityDescription(
+            key=DPCode.WORK_STATE,
+            name="Mode",
+            entity_category=EntityCategory.DIAGNOSTIC,
+            translation_key="work_state",
+        ),
```

Comment:
Hmm, in most cases, this is not a read-only value, thus this shouldn't be a sensor?

Reply:
From when I have seen the `WORK_STATE` just updated when the is switched on

Reply:
So, what is the added value to have this sensor in this case? I'm confused?

Reply:
From that I have seen the WORK_STATE can not be set, it's just an informational detail about it's current working mode: Idle, Active...
What do you want me to change?

Reply:
I don't see why this is added at all. The workstate is used to provide information for other entities, it shouldn't be a sensor on its own?

Reply:
Ok I will remove this

Reply:
I have also removed "WORK_STATE" as it's not clear or at least I don't understand what is the actual implementation requirements